inc/upgrade.php

```diff
@@ -188,25 +207,56 @@ function responsive_upgrade_2_0( $verbose = true ) {
 		update_post_meta( $result->post_id, '_wp_page_template', $template_map[ $result->meta_value ] );
 	}
 
+	responsive_upgrade_banner( $verbose );
+	responsive_upgrade_layout( $verbose );
+
+	// Delete the sytles cache so we can rebuild it after upgrading fonts and colors.
+	responsive_flush_customizer_styles_cache();
+	responsive_upgrade_fonts( $verbose );
+	responsive_upgrade_colors( $verbose );
+
+	// Recreate the styles cache.
+	responsive_get_customizer_styles( false );
+
+	// Delete unnecessary options.
+	delete_option( 'burf_setting_color_scheme' );
+	delete_option( 'burf_setting_active_color_region' );
+	delete_option( 'burf_setting_custom_colors' );
+}
+
+/**
+ * Migrate banner information
+ *
+ * @param boolean $verbose Flag for outptting messages to error log.
```

Comment:
`outptting` => `outputting` 😄 
```suggestion
 * @param boolean $verbose Flag for outputting messages to error log.
```